tasks/taglib.rb

```diff
@@ -0,0 +1,45 @@
+# frozen-string-literal: true
+
+class Taglib
```

Comment:
nit: Maybe `s/Taglib/TagLib/`?

Reply:
Sorry for the meta question but what do you mean by "nit:"?

This [website](https://conventionalcomments.org/) suggests it means:

> Nitpicks are trivial preference-based requests. These should be non-blocking by nature.

If a request is truly trivial and non-blocking one wouldn't make it in the first place.

Reply:
I renamed the helper to `Build` because `TagLib` is already taken.

Reply:
Yeah, a nit is just a small thing that's not blocking the change but might be nice to get addressed. Like a nice to have. Something the author could say "No, I actually prefer it the other way" and that's fine.